src/components/appcontent.tsx

```diff
@@ -25,10 +25,12 @@ const AppContent: FC<Props> = ({ showSidePane, showNavbar, appContent }) => {
   }, [location?.pathname]);
 
   useEffect(() => {
-    const _selectedMenu = MENU_LIST.find(
-      (menu) => location?.pathname.split("/")[1] == menu.path.split("/")[1],
-    ).id;
-    setSelectedMenu(_selectedMenu);
+    if (location?.pathname.split("/")[1] !== "orderDetails") {
```

Comment:
why this if loop ?

Reply:
For the order details page, we don't have any menu to select.